statsig/layer.py

```diff
@@ -31,6 +31,13 @@ def get(self, key, default=None):
 
         return default
 
+    def get_value(self):
```

Comment:
You use case for layers is pretty different than the general use case.  We don't want to expose this method generally on the SDK, because it can easily lead to misuse in the general case, and over exposure for other people integrating.

You should be able to write this sort of helper yourself, and reach in to the layer class to return the entire value.  But we dont want to add this functionality generally to the SDK

Reply:
Ok that's fine. By reach in to the layer, you mean to use the private `__value` property?

Reply:
Yeah.  We'd rather not add this to the SDK, but you should be able to access that field for now to get things going